### DISCUSSION TITLE
mkdocs 9.0.0 breaks our docs build - can only join an iterable?

### DISCUSSION THREAD

**Discussion by planetf1** [squidfunk/mkdocs-material#4816] [Q&A] [answered]
We noticed our github actions failing as of yesterday - with the change of mkdocs-material from 8.5.3 to 9.0.0

The first error is:
```
ERROR    -  Error building page 'release-notes/1-0.md': can only join an iterable
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.11.1/x64/bin/mkdocs", line 8, in <module>
    sys.exit(cli())
             ^^^^^
```
* our local issue : https://github.com/odpi/egeria-docs/issues/653
* Example action : https://github.com/odpi/egeria-docs/actions/runs/3823032364
* workflow : https://github.com/odpi/egeria-docs/blob/main/.github/workflows/pr-mkdocs.yml

I get the same locally (macOS, python 3.10.9) -- Just in the process of trying to figure out cause/strip down the configuration as per the troubleshooting guide - any thoughts about that specific issue welcome!

**Comment by 1871fotc** [comment]
Same here. Also trying to deduce cause.

**Reply by squidfunk** [reply]
We need a reproduction to look into it.

**Reply by planetf1** [reply]
Sure - I think I've managed that now. I opened up #4817 since I think I have enough details now? Hope it helps. Seems to relate to numeric filenames?

**Comment by planetf1** [comment, marked as answer]
See https://github.com/squidfunk/mkdocs-material/issues/4817 - I believe the cause relates to the use of numeric filenames - also referenced in mkdocs.yml

**Reply by squidfunk** [reply]
The issue was fixed.
